go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/mattermost/mattermost-plugin-channel-export
 
-go 1.14
+go 1.16
```

Comment:
In a separate PR, yes. This is minimal change required. I don't want to trigger forced upgrades of unrelated packages in this PR.  I had to bump to 1.16, otherwise I wouldn't have changed it at all.